src/components/Rocket.js

```diff
@@ -0,0 +1,34 @@
+import { useDispatch } from 'react-redux';
```

Comment:
According to the project requirements, you should use `useSelector()` to get a slice of the `store` in the component. Please fix this.

<img width="499" alt="Screen Shot 2022-08-26 at 8 21 18" src="https://user-images.githubusercontent.com/82426682/186913122-1209151f-b72a-4204-b212-a06b6b6cceb2.png">